lib/internal/process.js

```diff
@@ -9,6 +9,13 @@ function lazyConstants() {
   return _lazyConstants;
 }
 
+var _errors = null;
+function lazyErrors() {
+  if (!_errors)
+    _errors = require('internal/errors');
+  return _errors;
+}
```

Comment:
this makes the third way I've seen `internal/errors` loaded. I think this can be safely loaded immediately w/o fear of performance loss and be rid of all the lazy loading magic.